docs/guidelines-reviewee.md

```diff
@@ -0,0 +1,27 @@
+# Guidelines for reviewees
+
+This document contains some suggested guidelines and ideas for reviewees.
+A reviewee is someone who puts their code forward for review by others.
+
+In group discussions about the review process, the following have been chosen as good guidelines for reviewees to follow.
+
+## Good Practice
+
+This list gives high-level advice about good practice when putting your code forward for review.
+
+Please avoid controversial or highly-opinionated suggestions; there's a second list for more opinionated items below.
+
+If it looks like two items in this list contradict each other, the contradiction should be resolved, or they should be moved to the opinionated list below.
```

Comment:
These are a little bit meta; they're instructions for people editing the document, not instructions for reviewees. Could this be moved above the 'good practice' subheading, to make it clearer?

Reply:
Yes, that's a nice point; thank you. I've made amendments.